sensors/imu/lsm9dsxx.c

```diff
@@ -236,6 +241,13 @@ static void lsm9dsxx_threadPublish(void *data)
 			ctx->evtGyro.gyro.gyroY = translateGyr(ibuf[3], ibuf[2]);
 			ctx->evtGyro.gyro.gyroZ = translateGyr(ibuf[5], ibuf[4]);
 			ctx->evtGyro.timestamp = tstamp_gyro;
+
+			/* Integration of current measurement */
+			step = (tstamp_gyro - lastGyroTime) / 1000;
+			ctx->evtGyro.gyro.dAngleX = dAngleX = dAngleX + ctx->evtGyro.gyro.gyroX * step;
+			ctx->evtGyro.gyro.dAngleY = dAngleY = dAngleY + ctx->evtGyro.gyro.gyroY * step;
+			ctx->evtGyro.gyro.dAngleZ = dAngleZ = dAngleZ + ctx->evtGyro.gyro.gyroZ * step;
```

Comment:
MISRA rule 13.4 violation: The result of an assignment operator should not be used

Reply:
Oh... thought it is compliant. Thanks for pointing it out!